docs/tidb_http_api.md

```diff
@@ -477,3 +477,18 @@ timezone.*
     * op=nowait: return after binlog status is recoverd, do not wait until the skipped-binlog transactions are committed.
     * op=reset: reset `SkippedCommitterCounter` to 0 to avoid the problem that `SkippedCommitterCounter` is not cleared due to some unusual cases.
     * op=status: Get the current status of binlog recovery.
+
+1. Enable/disable async commit feature
```

Comment:
This may cause a security hazard. Previously, this kind of APIs affected at most one tidb-server.

Reply:
What security risk can there be if we provide such an API with very restricted function?

Reply:
It bypasses the privilege check. I don't think it's necessary to do this for me. If it is really necessary, some risk is acceptable.

Reply:
OK, I know the reason.